sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificatesTestBase.cs

```diff
@@ -42,6 +42,8 @@ public CertificatesTestBase(bool isAsync, CertificateClientOptions.ServiceVersio
             : base(isAsync, mode)
         {
             _serviceVersion = serviceVersion;
+            // temporary until https://github.com/Azure/azure-sdk-for-net/issues/27688 is addressed
```

Comment:
Based on the history of this and the KeysTestBase.cs, it's probably fine to still compare bodies despite our offline conversation. Maybe I fixed the problem I mentioned a while ago. But I do know there's a few other libraries I worked on / own that I set this to `false` because of floating point serialization issues between .NET Framework and .NET (Core).